infoFileComparison/src/main/scala/za/co/absa/hermes/infoFileComparison/AtumModelUtils.scala

```diff
@@ -142,8 +144,23 @@ object AtumModelUtils {
           checkpointValue.compareWith(otherCheckpoint.controls(index), nextPath) ::: agg
       }
 
+      val wasSoftware = checkpoint.software.getOrElse("Not Specified")
+      val isSoftware = otherCheckpoint.software.getOrElse("Not Specified")
+      val wasVersion = checkpoint.version.getOrElse("Not Specified")
+      val isVersion = otherCheckpoint.version.getOrElse("Not Specified")
+
+      logCheckpointDiff(wasSoftware, wasVersion, isSoftware, isVersion)
+
       diffs.flatten ::: controls
+
     }
+
+    private def logCheckpointDiff (refSoftware: String, refVersion: String,
+                                   newSoftware: String, newVersion: String): Unit = {
+      if (refSoftware != newSoftware) scribe.info(s"Software was - $refSoftware and is - $newSoftware.")
```

Comment:
- is the message enough? try to imaging seeing "Software was - alfa and is - beta" in the middle of all logs.
- I think it should be a warning. A warning would tell me as a user that something isn't 100% but it doesn't have to be an issue.
- I would use only one `if` and print both if any differ. Would make for a more concise message.